### DISCUSSION TITLE
Endpoint response in swagger docs even without closing braces

### DISCUSSION THREAD

**Discussion by dunossauro** [fastapi/fastapi#8451] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [x] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from pydantic import BaseModel


class Response(BaseModel):
    message: str


app = FastAPI()


@app.get('/user/{_id', response_model=Response)  # NOTE: The brace is open but not closed 
def user(_id: int):
    return {'message': f'Olá {_id}'}
```

### Description

Endpoint response in swagger docs even without closing braces.

## Swagger test:
![image](https://user-images.githubusercontent.com/6801122/161063033-62f2f42f-3120-43b2-86fa-e0375367fcf1.png)

The same behavior does not occur in calls made by httpie or curl 

## Httpie tests:
```
http :8000/user/1
HTTP/1.1 404 Not Found
content-length: 22
content-type: application/json
date: Thu, 31 Mar 2022 13:12:18 GMT
server: uvicorn

{
    "detail": "Not Found"
}
```

### Operating System

Linux

### FastAPI Version

0.73.0

### Python Version

3.10

**Comment by wmcgee3** [comment, marked as answer]
It looks like the path is literally `/user/{_id` because of the lack of closing brace and the Swagger docs are including `_id` as a query parameter.

The Curl and Request URL in your screenshot are looking a little gnarly!

**Comment by jonatasoli** [comment]
The /user/{_id was interpreted as part of the url so your test in httpie using this url:
/user/{_id?_id=1 and no /user/1
